deployment/node-feature-discovery/templates/master.yaml

```diff
@@ -44,14 +44,17 @@ spec:
             - "nfd-master"
           resources:
             {{- toYaml .Values.master.resources | nindent 12 }}
+          args:
+            { { - if .Values.master.instance | empty | not } }
```

Comment:
will it works when .Values.master.instance is empty? won't you have args: with empty flags?

Reply:
It will pass empty flag, in this case no args will be rendered. Let's add empty value for consistency with orhter